cookbook/playground/upload_files.py

```diff
@@ -0,0 +1,41 @@
+from phi.agent import Agent
+from phi.knowledge.docx import DocxKnowledgeBase
+from phi.knowledge.json import JSONKnowledgeBase
+from phi.knowledge.pdf import PDFKnowledgeBase
+from phi.knowledge.csv import CSVKnowledgeBase
+from phi.knowledge.combined import CombinedKnowledgeBase
+from phi.knowledge.text import TextKnowledgeBase
+from phi.playground.playground import Playground
+from phi.playground.serve import serve_playground_app
+from phi.vectordb.pgvector import PgVector
+
+db_url = "postgresql+psycopg://ai:ai@localhost:5532/ai"
+
+knowledge_base = CombinedKnowledgeBase(
```

Comment:
Why the knowledgebases? The uploaded file isn't added here right?

Reply:
That is how CombinedKnowledgeBase works sadly. It requires list of sources which is Aent knowledge Bases